.github/workflows/release_prep.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o pipefail
+
+# Single tag arg is passed by https://github.com/bazel-contrib/.github/blob/master/.github/workflows/release_ruleset.yaml
+TAG="$1"
+# Assumes tag format vX.Y.Z, drops the leading 'v'
```

Comment:
Let's verify that assumption and report an error if it doesn't start with `v`. `release.yaml` doesn't check this for the `workflow_dispatch` trigger.